### PR TITLE
Added ssl client verification

### PR DESCRIPTION
**Pull request by manucarrace**

### CODE REVIEW

File: iotqatools/cb_ngsiv2_utils.py [hunk 1]
```diff
@@ -260,6 +263,9 @@ def __send_request(self, method, url, headers=None, payload=None, verify=None, q
 
         if verify is not None:
             parameters.update({'verify': verify})
+        else:
+            # If the method does not include the verify parameter, it takes the vale from object
```
Comment (fgalan) [inline iotqatools/cb_ngsiv2_utils.py]: Typo: vale -> value

File: iotqatools/cep_utils.py [hunk 2]
```diff
@@ -219,16 +262,17 @@ def create_visual_rule(self, rule_name, service, sensor_card_list, action_card_l
...
-        print url
-        print json.dumps(yaml.load(cep_payload))
-        response = requests.post(url, data=json.dumps(yaml.load(cep_payload)), headers=headers, verify=False)
-        assert response.status_code == 201, 'ERROR {}, the rule {} cannot be created. {}'.format(response.status_code,
-                                                                                                 rule_name,
-                                                                                                 response.text)
-        return response
+        return self.__send_request('post', url, payload=json.dumps(yaml.load(cep_payload)), headers=headers,
+                                   verify=self.verify)
+#        response = requests.post(url, data=json.dumps(yaml.load(cep_payload)), headers=headers, verify=False)
```
Comment (fgalan) [inline iotqatools/cep_utils.py]: Leftover block of code to be removed?

Comment (fgalan) [reply iotqatools/cep_utils.py]: More similar cases in the same file.

File: iotqatools/orchestator_utils.py [hunk 3]
```diff
@@ -61,6 +63,11 @@ def send(self, method, url, headers=None, payload=None, query=None):
             request_parms.update({'data': payload})
         if query is not None:
             request_parms.update({'params': query})
+        if verify is not None:
+            request_parms.update({'verify': verify})
+        else:
+            # If the method does not include the verify parameter, it takes the vale from object
```
Comment (fgalan) [inline iotqatools/orchestator_utils.py]: Typo: vale -> value

File: iotqatools/sth_utils.py [hunk 4]
```diff
@@ -98,6 +101,10 @@ def __send_request(self, method, url, headers=None, payload=None, verify=None, q
 
         if verify is not None:
             parameters.update({'verify': verify})
+        else:
+            # If the method does not include the verify parameter, it takes the vale from object
```
Comment (fgalan) [inline iotqatools/sth_utils.py]: Typo: vale -> value

File: iotqatools/cep_utils.py (hunk 2 above)
Comment (manucarrace) [reply iotqatools/cep_utils.py]: Removed in 773742c

File: iotqatools/cb_ngsiv2_utils.py (hunk 1 above)
Comment (manucarrace) [reply iotqatools/cb_ngsiv2_utils.py]: Fixed in 3b6e677

File: iotqatools/orchestator_utils.py (hunk 3 above)
Comment (manucarrace) [reply iotqatools/orchestator_utils.py]: Fixed in 3b6e677

File: iotqatools/sth_utils.py (hunk 4 above)
Comment (manucarrace) [reply iotqatools/sth_utils.py]: Fixed in 3b6e677

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**